.github/workflows/push.yml

```diff
@@ -19,22 +19,19 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: ['2.5.1', '2.7.2', '2.6.5']
+        ruby: ['3.2.2']
         gemfile:
           - gemfiles/ruby_kafka.gemfile
-          - gemfiles/que_0.12.2.gemfile
-          - gemfiles/que_0.12.3.gemfile
-          - gemfiles/rails_5.2.gemfile
-          - gemfiles/shoryuken_4.0.gemfile
-          - gemfiles/sidekiq_4.2.gemfile
-          - gemfiles/sinatra_1.4.gemfile
-          - gemfiles/sinatra_2.0.gemfile
```

Comment:
Did all of those fail? If not, id say we leave the most reasonable ones still active!? That way we can ensure that some older versions are still running without major issues.